datetime_formatter/models/res_lang.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2015, 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+# Copyright 2016 Tecnativa, S.L. - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from datetime import datetime, timedelta
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT, DEFAULT_SERVER_TIME_FORMAT
+
+# Available modes for :param:`.ResLang.datetime_formatter.template`
+MODE_DATETIME = "MODE_DATETIME"
+MODE_DATE = "MODE_DATE"
+MODE_TIME = "MODE_TIME"
+
+
+class ResLang(models.Model):
+    _inherit = "res.lang"
+
+    @api.model
+    @api.returns("self")
```

Comment:
I think this is not necessary. The migration steps specify that this decorator can be removed, and besides it returns "self", so I think it can be removed:
```suggestion
```